Client/Client.Core/Client.Core.Static/Client.cpp

```diff
@@ -329,14 +329,14 @@ void Client::loop()
 
             case MessageType::ChannelListRequest:
             {
-                auto channels = std::any_cast<std::vector<Network::ChannelInfo>>(message.mBody);
+                auto channels = std::any_cast<std::vector<Base::Models::ChannelInfo>>(message.mBody);
```

Comment:
You could apply Models namespace here.
This should sort the length of models name. 
For example: Models::ChannelInfo

Reply:
Done